app/views/layouts/application.html.erb

```diff
@@ -3,32 +3,46 @@
   <head>
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <title><%= content_for?(:title) ? yield(:title) : "DARIA" %></title>
-    <meta name="description" content="<%= content_for?(:description) ? yield(:description) : "DCAF" %>">
+    <meta name="description" content="<%= content_for?(:description) ? yield(:description) : "DARIA" %>">
     <meta name='ROBOTS' content='NOINDEX, NOFOLLOW'>
     <meta name="google-site-verification" content="<%= ENV['GOOGLE_SITE_VERIFICATION'] %>" />
 
-    <%= csrf_meta_tags %>
+    <% if !content_for? :error_page %>
```

Comment:
With some gentle control flow here we can reuse the main layout. Essentially, the reasoning is: If there was an error, no need to set up CSRF, render async, or flash messages.